proto/redpanda/api/dataplane/v1alpha2/pipeline.proto

```diff
@@ -378,3 +447,9 @@ message StartPipelineRequest {
 message StartPipelineResponse {
   Pipeline pipeline = 1;
 }
+
+message GetPipelineServiceConfigSchemaRequest {}
+
+message GetPipelineServiceConfigSchemaResponse {
+  string config_schema = 1;
```

Comment:
nit: comments on what this is would be helpful to future readers. Maybe something like:

```
// config_schema is the JSON schema of the configuration files that are allowed for pipelines.
```